.idea/workspace.xml

```diff
@@ -0,0 +1,291 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Wat

Reply:
uh lol this is something jetbrains adds that needs to be gitignored

Reply:
lol. If you didn’t know already you can Ignore files in git at the machine/user level. I’m on mobile, google it !

Reply:
yup i'm taking care of it on my end 👍